test/test.dart

```diff
@@ -320,10 +323,15 @@ Future<void> setupBaas() async {
   final privateApiKey = arguments[argBaasPrivateApiKey];
   final projectId = arguments[argBaasProjectId];
   final differentiator = arguments[argDifferentiator];
+  final jwksUrl = arguments[argJwksUrl];
 
   final client = await (cluster == null
       ? BaasClient.docker(baasUrl, differentiator)
       : BaasClient.atlas(baasUrl, cluster, apiKey!, privateApiKey!, projectId!, differentiator));
+  if (!isFlutterPlatform || (Platform.isWindows || Platform.isLinux)) {
```

Comment:
We need to find better location of these files with keys so that they could be accessed by Flutter Android, IOS, MacOS tests also. Currently flutter tests for these platforms don't run the tests for JWT, because of missing path "test/data/jwt_keys/".

Reply:
if we pass content on as fx argBaasApiKey we don't need to

Reply:
Removed.